lib/mvi/service.rb

```diff
@@ -58,27 +60,37 @@ def find_profile(user)
 
     private
 
-    # TODO: Possibly consider adding Grafana Instrumentation here too
     def mvi_error_handler(user, e)
       case e
       when MVI::Errors::DuplicateRecords
-        log_message_to_sentry('MVI Duplicate Record', :warn)
+        log_info_and_errors('MVI Duplicate Record', :warn)
       when MVI::Errors::RecordNotFound
-        # Not going to log RecordNotFound
-        # NOTE: ICN based lookups do not return RecordNotFound. They return InvalidRequestError
-        # log_message_to_sentry('MVI Record Not Found', :warn)
-        nil
+        # Not going to log RecordNotFound to sentry, cloudwatch only.
+        log_info_and_errors('MVI Record Not Found')
       when MVI::Errors::InvalidRequestError
+        # NOTE: ICN based lookups do not return RecordNotFound. They return InvalidRequestError
         if user.mhv_icn.present?
-          log_message_to_sentry('MVI Invalid Request (Possible RecordNotFound)', :error)
+          log_info_and_errors('MVI Invalid Request (Possible RecordNotFound)', :error)
         else
-          log_message_to_sentry('MVI Invalid Request', :error)
+          log_info_and_errors('MVI Invalid Request', :error)
         end
       when MVI::Errors::FailedRequestError
-        log_message_to_sentry('MVI Failed Request', :error)
+        log_info_and_errors('MVI Failed Request', :error)
       end
     end
 
+    def log_info_and_errors(message, sentry_classification = nil)
+      Rails.logger.debug(message)
```

Comment:
I think we're going to want info level messages here so we maintain these in the production logs

Reply:
are you sure the debug ones don't trickle up? see the examples  below that I tailed from `log/test.log`